doc/tutorials/klondike/app/lib/step3/klondike_game.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:math';
+
+import 'package:flame/components.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/game.dart';
+
+import 'components/card.dart';
+import 'components/foundation.dart';
+import 'components/pile.dart';
+import 'components/stock.dart';
+import 'components/waste.dart';
+
+class KlondikeGame extends FlameGame {
+  static const double cardGap = 175.0;
+  static const double cardWidth = 1000.0;
+  static const double cardHeight = 1400.0;
+  static const double cardRadius = 100.0;
+  static final Vector2 cardSize = Vector2(cardWidth, cardHeight);
+
+  @override
+  Future<void> onLoad() async {
+    await Flame.images.load('klondike-sprites.png');
+
+    final stock = Stock()
+      ..size = cardSize
+      ..position = Vector2(cardGap, cardGap);
+    final waste = Waste()
+      ..size = cardSize
+      ..position = Vector2(cardWidth + 2 * cardGap, cardGap);
+    final foundations = List.generate(
+      4,
+      (i) => Foundation()
+        ..size = cardSize
+        ..position =
+            Vector2((i + 3) * (cardWidth + cardGap) + cardGap, cardGap),
+    );
+    final piles = List.generate(
+      7,
+      (i) => Pile()
+        ..size = cardSize
+        ..position = Vector2(
+          cardGap + i * (cardWidth + cardGap),
+          cardHeight + 2 * cardGap,
+        ),
+    );
+
+    final world = World()
+      ..add(stock)
+      ..add(waste)
+      ..addAll(foundations)
+      ..addAll(piles);
+    add(world);
+
+    final camera = CameraComponent(world: world)
+      ..viewfinder.visibleGameSize =
+          Vector2(cardWidth * 7 + cardGap * 8, 4 * cardHeight + 3 * cardGap)
+      ..viewfinder.position = Vector2(cardWidth * 3.5 + cardGap * 4, 0)
+      ..viewfinder.anchor = Anchor.topCenter;
+    add(camera);
+
+    final random = Random();
+    for (var i = 0; i < 7; i++) {
+      for (var j = 0; j < 4; j++) {
+        final card = Card(random.nextInt(13) + 1, random.nextInt(4))
```

Comment:
This is a temporary code that adds the cards to the world in order to be able to see how they're rendered. Since there isn't enough space to see all 52 cards at once, I rely on randomness + page reloads to see them all. This may not be ideal, but this code will be removed in the next chapter anyways when we start putting cards where they actually belong.